modules/kafka/kafka-dest-driver.c

```diff
@@ -606,16 +661,9 @@ kafka_dd_free(LogPipe *d)
   KafkaDestDriver *self = (KafkaDestDriver *)d;
 
   log_template_options_destroy(&self->template_options);
-  if (self->topics)
-    g_hash_table_unref(self->topics);
+  _destroy_kafka(&self->super.super.super);
   log_template_unref(self->key);
   log_template_unref(self->message);
-  if (self->topic)
-    rd_kafka_topic_destroy(self->topic);
-  if (self->fallback_topic_name)
-    g_free(self->fallback_topic_name);
```

Comment:
fallback_topic_name should be free up, right?

Reply:
fixed.